packages/yoastseo/src/languageProcessing/helpers/sentence/getSentences.js

```diff
@@ -25,16 +23,15 @@ const newLineRegex = new RegExp( newLines );
  * @param {string} block The HTML inside a HTML block.
  * @returns {Array<string>} The list of sentences in the block.
  */
-function getSentencesFromBlock( block ) {
-	const sentenceTokenizer = isFeatureEnabled( "JAPANESE_SUPPORT" ) ? new JapaneseSentenceTokenizer() : new SentenceTokenizer();
+function getSentenceTokenizer( block ) {
+	const sentenceTokenizer = new JapaneseSentenceTokenizer();
```

Comment:
@agnieszkaszuba I think the `JapaneseSentenceTokenizer` should be renamed to `SentenceTokenizer`